google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/v1/MessageDispatcherTest.java

```diff
@@ -139,6 +139,16 @@ public void testReceiptMessageReceiver() {
         .receiveMessage(eq(TEST_MESSAGE.getMessage()), any(AckReplyConsumer.class));
   }
 
+  // @Test
```

Comment:
Remove this?

But, let's see if we can add a test for the change in behavior for exactly once?